.github/workflows/ci.yml

```
@@ -77,92 +158,185 @@ jobs:
       run: |
         if (-Not (Test-Path C:\SDL))
         {
-            Invoke-WebRequest "https://github.com/libsdl-org/SDL/releases/download/release-$env:SDL_VERSION/SDL2-devel-$env:SDL_VERSION-VC.zip" -OutFile C:\SDL.zip
+            Invoke-WebRequest "https://github.com/libsdl-org/SDL/releases/download/release-$env:SDL2_VERSION/SDL2-devel-$env:SDL2_VERSION-VC.zip" -OutFile C:\SDL.zip
             Expand-Archive C:\SDL.zip -DestinationPath C:\
         }
 
     - name: CMake configure (Debug)
       run: |
-        $env:LDFLAGS =  "/LIBPATH:C:\SDL2-$env:SDL_VERSION\lib\x86 "
+        $env:LDFLAGS =  "/LIBPATH:C:\SDL2-$env:SDL2_VERSION\lib\x86 "
         cmake -B debug -G "Visual Studio 17 2022" . -DCMAKE_BUILD_TYPE=Debug `
           -A Win32 `
-          -DSDL2_INCLUDE_DIRS="C:\SDL2-$env:SDL_VERSION\include" `
+          -DSDL2_INCLUDE_DIRS="C:\SDL2-$env:SDL2_VERSION\include" `
           -DSDL2_LIBRARIES="SDL2;SDL2main"
 
     - name: Build (Debug)
       run: cmake --build debug
 
     - name: CMake configure (Release)
       run: |
-        $env:LDFLAGS =  "/LIBPATH:C:\SDL2-$env:SDL_VERSION\lib\x86 "
+        $env:LDFLAGS =  "/LIBPATH:C:\SDL2-$env:SDL2_VERSION\lib\x86 "
         cmake -B release -G "Visual Studio 17 2022" . -DCMAKE_BUILD_TYPE=Release `
           -A Win32 `
-          -DSDL2_INCLUDE_DIRS="C:\SDL2-$env:SDL_VERSION\include" `
+          -DSDL2_INCLUDE_DIRS="C:\SDL2-$env:SDL2_VERSION\include" `
           -DSDL2_LIBRARIES="SDL2;SDL2main"
 
     - name: Build (Release)
       run: cmake --build release
 
-  windows-mingw:
-    name: Windows (MinGW)
-    runs-on: windows-latest
-    defaults:
-      run:
-        shell: msys2 {0}
+  freebsd:
+    runs-on: ubuntu-latest
+    name: FreeBSD
     steps:
-    - uses: actions/checkout@v3
+    - uses: actions/checkout@v4
       with:
         submodules: true
+    - name: Build
+      uses: cross-platform-actions/action@v0.24.0
+      with:
+        operating_system: freebsd
+        version: '13.3'
+        run: |
+          sudo pkg update
+          sudo pkg install -y cmake ninja sdl2
+          cmake -B build -G Ninja .
+          cmake --build build --verbose -- -j`sysctl -n hw.ncpu`
+
+  linux-SDL3:
+    name: Rocky Linux SDL3
+    runs-on: ubuntu-latest
+    container:
+      image: rockylinux/rockylinux:8
+    steps:
+    - name: Are we really on Rocky Linux?
+      run: cat /etc/os-release
+
+    - name: Install dependencies
+      run: |
+        dnf update -y
+        dnf install -y epel-release
+        dnf config-manager --set-enabled powertools
+        dnf install -y bzip2 bzip2-devel chrpath clang cmake freetype-devel gcc-c++ \
+            git gtk3-devel libstdc++-static libuuid-devel libvorbis-devel \
+            libxml2-devel meson ninja-build ocl-icd-devel openal-soft-devel \
+            opencl-headers openssl-devel patch perl-IPC-Cmd svn unix2dos xz zlib-static
```

Cleaning up my own mess:

```suggestion
        dnf install -y chrpath cmake gcc-c++ git ninja-build
```